apps/search/index.html

```diff
@@ -93,10 +93,15 @@ <h2 id="places-header" class="hidden"><span data-l10n-id="history"></span></h2>
                aria-label="Places"></section>
 
       <div id="suggestions-wrapper">
-        <h2><span id="suggestion-provider"></span></h2>
+        <h2>
+          <select id="suggestions-select"></select>
+          <label for="suggestions-select" id="suggestions-provider"></label>
+          <div class="link-triangle"> </div>
```

Comment:
Its that little triangle on the bottom right to make it look a bit more like a select box